tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+	"compilerOptions": {
+		"module": "umd",
+		"noImplicitAny": true,
+		"removeComments": true,
+		"preserveConstEnums": true,
+		"sourceMap": true,
+		"target": "es5"
```

Comment:
It's that we can't make implicit nullables, right (based on the explanation)? If so I' for it as well.